docs/noencap-hybrid-modes.md

```diff
@@ -0,0 +1,174 @@
+Antrea might be unnecessary. In this case, you can disable it by setting the
+`noSNAT` config option to `true`. The `trafficEncapMode` and `noSNAT` config
+options are defined in `antrea-agent.conf` of the `antrea` ConfigMap in the
+[Antrea deployment YAML](https://github.com/vmware-tanzu/antrea/blob/main/build/yamls/antrea.yml).
+
+```yaml
+  antrea-agent.conf: |
+    ... ...
+    # Determines how traffic is encapsulated. It has the following options:
+    # encap(default):    Inter-node Pod traffic is always encapsulated and Pod to external network
+    #                    traffic is SNAT'd.
+    # noEncap:           Inter-node Pod traffic is not encapsulated; Pod to external network traffic is
+    #                    SNAT'd if noSNAT is not set to true. Underlying network must be capable of
+    #                    supporting Pod traffic across IP subnets.
+    # hybrid:            noEncap if source and destination Nodes are on the same subnet, otherwise encap.
+    # networkPolicyOnly: Antrea enforces NetworkPolicy only, and utilizes CNI chaining and delegates Pod
+    #                    IPAM and connectivity to the primary CNI.
+    #
+    trafficEncapMode: noEncap
+
+    # Whether or not to SNAT (using the Node IP) the egress traffic from a Pod to the external network.
+    # This option is for the noEncap traffic mode only, and the default value is false. In the noEncap
+    # mode, if the cluster's Pod CIDR is reachable from the external network, then the Pod traffic to
+    # the external network needs not be SNAT'd. In the networkPolicyOnly mode, antrea-agent never
+    # performs SNAT and this option will be ignored; for other modes it must be set to false.
+    #noSNAT: false
+    ... ...
+```
+
+After changing the options, you can deploy Antrea with the `noEncap` mode by
+applying the deployment YAML.
+
+### Using kube-router for BGP
+
+We can run kube-router in the advertisement-only mode to advertise Pod CIDRs to
+the peered routers, so the routers can know how to route Pod traffic to the
+Nodes. To deploy kube-router in the advertisement-only, first download the
+[kube-router DaemonSet template](https://raw.githubusercontent.com/cloudnativelabs/kube-router/v0.4.0/daemonset/generic-kuberouter-only-advertise-routes.yaml):
+
+```bash
+curl -LO https://raw.githubusercontent.com/cloudnativelabs/kube-router/v0.4.0/daemonset/generic-kuberouter-only-advertise-routes.yaml
+```
+
+Then edit the YAML file and set the following kube-router arguments:
+
+```yaml
+- "--run-router=true"
+- "--run-firewall=false"
+- "--run-service-proxy=false"
+- "--enable-cni=false"
+- "--enable-ibgp=false"
+- "--enable-overlay=false"
+- "--enable-pod-egress=false"
+- "--peer-router-ips=<CHANGE ME>"
+- "--peer-router-asns=<CHANGE ME>"
+```
+
+The BGP peers should be configured by specifying the --peer-router-asns and
+--peer-router-ips parameters. Note, the ASNs and IPs must match the
```

Comment:
nit: use back ticks around `--peer-router-asns` and `--peer-router-ips`

Reply:
Done.